setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Function to download a file if it does not exist
+download_file() {
+  local url=$1
+  local target_path=$2
+
+  # Check if the file already exists to avoid re-downloading
+  if [ ! -f "$target_path" ]; then
+    echo "Downloading $(basename "$target_path")..."
+    curl -s -L "$url" -o "$target_path" || { echo "Failed to download $(basename "$target_path"). Exiting."; exit 1; }
+  else
+    echo "$(basename "$target_path") already exists. Skipping download."
+  fi
+}
+
+# Set databases path
+GRIDSUITE_DATABASES=$(realpath "DATABASES")
+export GRIDSUITE_DATABASES
+
+# Create the subdirectories with the required file mode
+mkdir -p "$GRIDSUITE_DATABASES/cases" "$GRIDSUITE_DATABASES/postgres" "$GRIDSUITE_DATABASES/elasticsearch" "$GRIDSUITE_DATABASES/init" || { echo "Failed to create directories. Exiting."; exit 1; }
+
+# Change the file mode to 777 for all subdirectories
+chmod 777 "$GRIDSUITE_DATABASES/cases" "$GRIDSUITE_DATABASES/postgres" "$GRIDSUITE_DATABASES/elasticsearch" "$GRIDSUITE_DATABASES/init" || { echo "Failed to set permissions. Exiting."; exit 1; }
+
+echo "Folder structure created under $GRIDSUITE_DATABASES with required permissions."
+
+# Populate init folder
+download_file "https://raw.githubusercontent.com/gridsuite/geo-data/main/src/test/resources/geo_data_substations.json" "$GRIDSUITE_DATABASES/init/geo_data_substations.json"
+download_file "https://raw.githubusercontent.com/gridsuite/geo-data/main/src/test/resources/geo_data_lines.json" "$GRIDSUITE_DATABASES/init/geo_data_lines.json"
+download_file "https://raw.githubusercontent.com/gridsuite/cgmes-boundary-server/main/src/test/resources/business_processes.json" "$GRIDSUITE_DATABASES/init/business_processes.json"
+download_file "https://raw.githubusercontent.com/gridsuite/cgmes-boundary-server/main/src/test/resources/tsos.json" "$GRIDSUITE_DATABASES/init/tsos.json"
```

Comment:
It's already done in Postgres init script  
https://github.com/gridsuite/deployment/blob/fda0ebdbf8eec7f533a765067e0b16fd76b05d37/docker-compose/technical/init-databases.sh#L7-L8

Reply:
you need to download them to init folder manually currently, then from init folder the actual in docker container database is initialized with the scripts you referred